SSO-Auth/Api/SSOController.cs

```diff
@@ -90,10 +90,10 @@ public SSOController(ILogger<SSOController> logger, ISessionManager sessionManag
                 Scope = string.Join(" ", config.OidScopes.Prepend("openid profile")),
             };
             options.Policy.Discovery.ValidateEndpoints = false; // For Google and other providers with different endpoints
-            options.Policy.Discovery.RequireHttps = config?.RequireHttps ?? true;
+            options.Policy.Discovery.RequireHttps = config.RequireHttps;
```

Comment:
What if this isn't set?

Reply:
So a boolean type defaults to ```false``` in c#.

RequireHttps type is ```bool``` which isn't nullable. If it's type was ```bool?``` it could be null but that's not the case here.. 

If the config object was null then it would have thrown an error before we get here. So essentially the null check was redundant and thus the ?? pointless too.